dotCMS/src/main/java/com/dotmarketing/cms/comment/action/CommentsAction.java

```diff
@@ -65,7 +66,8 @@
  * @version 1.6
  * @since 1.0
  */
-public class CommentsAction extends DispatchAction {
+@Deprecated
+public class CommentsAction extends SecureAction {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)